ingest/ingest_pipeline.py

```diff
@@ -308,9 +285,22 @@ def create_parser():
         <gene file path>. See --help for more information"
 
     parser_ingest_expression.add_argument(
-        "--file-params",
-        type=json.loads,
-        help="Optional parameters for expression files",
+        "--taxon-name",
+        help="Taxon (scientific) name associated with file. Ex. 'Homo Sapiens' ",
+    )
+    parser_ingest_expression.add_argument(
+        "--taxon-common-name",
+        help="Taxon common name associated with study. Ex. 'humans'",
+    )
+    parser_ingest_expression.add_argument(
+        "--ncbi-taxid", help="NCBI ID associated with file"
+    )
+    parser_ingest_expression.add_argument(
+        "--genome-assembly-accession", help="Genome assembly accession for file."
```

Comment:
Examples are given in related arguments' `help` field (which is excellent), so for consistency:

```suggestion
        "--genome-assembly-accession", help="Genome assembly accession for file.  E.g. 'GCA_000001405.15'"
```